src/ui/action-button/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { memo } from 'react';
+import { Link } from 'react-router-dom';
+
+import Spinner from '../spinner';
+
+interface ActionButtonProps {
+  title: string;
+  path: string;
```

Comment:
```suggestion
  path?: string;
```

То же самое и относится к `disabled` и `isLoading`. Это необязательные пропсы, ниже я покажу пример